typescript/sdk/src/gas/HyperlaneIgpDeployer.ts

```diff
@@ -126,84 +109,73 @@ export class HyperlaneIgpDeployer extends HyperlaneDeployer<
           desiredGasData.tokenExchangeRate,
         )
       ) {
-        this.logger(
-          `${chain} -> ${remote} existing gas data:\n`,
-          prettyRemoteGasData(remoteGasDataConfig),
-        );
-        this.logger(
-          `${chain} -> ${remote} desired gas data:\n`,
-          prettyRemoteGasData(desiredGasData),
-        );
-        configsToSet[gasOracleAddress].push({
+        configsToSet.push({
```

Comment:
this is a misleading oversimplication of the configuring step where you're assuming you can't have different sets of gas oracles configured for different domains, like it will get updated in the new/wrong gas oracle whereas the onchain storage oracle will remain the same.

Reply:
it is not possible for our tooling to configure multiple gas oracles and the complexity here is unnecessary for all of the existing usage